pkg/storaged/filesystem/utils.jsx

```diff
@@ -185,9 +195,13 @@ export const mount_point_text = (mount_point, mounted) => {
         mp_text = client.strip_mount_point_prefix(mount_point);
         if (mp_text == false)
             return null;
-        if (!mounted)
+        if (!mounted && !client.in_anaconda_mode())
             mp_text = mp_text + " " + _("(not mounted)");
-    } else
-        mp_text = _("(not mounted)");
+    } else {
+        if (client.in_anaconda_mode())
+            mp_text = _("(no assigned mount point)");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19807-20240110-085306-03cea355-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)